### DISCUSSION TITLE
Gatsby + Joomla

### DISCUSSION THREAD

**Discussion by rikbrown1** [gatsbyjs/gatsby#29107] [Help]
We operate a Joomla site with 1.7 million dynamic pages. I would love to convert those pages to static pages with Gatsby. 

I viewed a Gatsby YouTube video discussing sites of 100k pages. Would generating a multi-million page site be possible with Gatsby? 

We would only need to regenerate every month or two. Pages typically have only one image.

Thank you for advising on the feasibility of this and any kind of estimate of time to generate.

**Comment by KyleAMathews** [comment]
It's possible. You'd need a lot of RAM & patience. Build time right now would probably be on the order of hours.

**Comment by KyleAMathews** [comment]
A bottleneck too is getting data out of Joomla — some sort of bulk export that Gatsby can then suck in would speed things up
